test/functional/NativeTest/playlist.xml

```diff
@@ -98,11 +88,33 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
-		<subsets>
-			<subset>8</subset>
-			<subset>9</subset>
-			<subset>11+</subset>
-		</subsets>
+	</test>
+	<test>
+		<testCaseName>jniargtestssystemlink</testCaseName>
+		<variations>
+			<variation>-Xint</variation>
+			<variation>-Xjit:count=0</variation>
+			<variation>-Xjit:count=1,disableAsyncCompilation</variation>
+		</variations>
+    <platformRequirements>os.zos,bits.31</platformRequirements>
```

Comment:
Please fix the indentation. The rest test change looks good to me.

Reply:
Fixed as part of the [force push](https://github.com/eclipse/openj9/compare/ab713cac6f91859325f0e62136936b0bb7ddbec8..81d53045bad70d344ef760e0a6b101cbae5d14f4) to rebase.